src/shadowbox/server/server_access_key.ts

```diff
@@ -106,6 +106,10 @@ function isValidCipher(cipher: string): boolean {
   return true;
 }
 
+function isValidPort(port: number): boolean {
```

Comment:
Use a [type predicate](https://www.typescriptlang.org/docs/handbook/2/narrowing.html#using-type-predicates):
```suggestion
function isValidPort(port: number): port is number {
```

Reply:
A type predicate doesn't really make sense here because we're not narrowing the type here: it's already a number.

Reply:
Duh, I got confused thinking about the HTTP service. I'm sorry about that!